Mage.Sets/src/mage/sets/fifthdawn/EonHub.java

```diff
@@ -40,7 +40,7 @@ public EonHub copy() {
     
 }
 
-class SkipUpkeepStepEffect extends ReplacementEffectImpl {
+class SkipUpkeepStepEffect extends ContinuousRuleModifiyingEffectImpl {
     
     public SkipUpkeepStepEffect() {
         super(Duration.WhileOnBattlefield, Outcome.Neutral);
```

Comment:
Check the now (now defaulted) log parameters. You probably won't have everytime a message to the player for this effect.